index.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  'all.schema-store': require('./schemas/all.schema-store.json'),
```

Comment:
Some would see this as a breaking change, or at least I think our parser does 🤔 Cause we are breaking the assumption that what is exposed here is a map of valid AsyncAPI versions.

https://github.com/asyncapi/parser-js/blob/c962a7e7278f5fc8652621d8881032fceaa79a28/lib/parser.js#L79

i.e. I wonder if the parser would accept:

```yml
asyncapi: "all.schema-store"
...
```

Reply:
This is now solved by not exporting this schema anymore. See https://github.com/asyncapi/spec-json-schemas/pull/237#discussion_r915695825